perftracker/templates/_base.html

```diff
@@ -81,14 +85,74 @@
                 </label>
                 <input type="password" class="form-control" id="exampleInputPassword1" placeholder="Password" />
               </div>
-              <button type="submit" class="btn btn-success btn-block" id='pt_btn_login'><i class="glyphicon glyphicon-ok"></i> Submit</button>
+              <button type="submit" class="btn btn-success btn-block" id='pt_btn_login' onclick="pt_auth_cb();"><i class="glyphicon glyphicon-ok"></i> Submit</button>
             </div>
           </li>
+
+          <li class="dropdown">
+            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false" onclick="pt_auth_cb();" id="sign_out_btn">
+              <i class="glyphicon glyphicon-user"></i> Sign Out
+            </a>
+          </li>
+
         </ul>
       </div><!-- /.navbar-header -->
     </div><!-- /.container-fluid -->
   </nav>
 
+  <script>
+  var api_ver = '{{ api_ver }}';
+  var project_id = {{ project.id }};
+  initialize_nav_bar();
+
+  $(function () {
+      $.ajax({
+          url: '/api/v{0}/{1}/auth'.ptFormat(api_ver, project_id),
+          data: null,
+          type: 'GET',
+          timeout: 2000,
+          success: function(data) {
+              update_nav_bar(data);
+          },
+          error: function(data, status, error) {
+              alert('Error: ' + xhr.status + ' ' + xhr.responseText);
```

Comment:
it can be annoying to get alerts on each page refresh, can we just report issue into login-related controls on the page?

Reply:
There are many ajax-requests in project and almost all of them use "alert()" in case of error. Are you sure that this case is so unique that I should use another way of reporting?